shrine/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
```

Comment:
We can have a launch screen created by the design team too.

Reply:
I still remember that animated diamond from the mock you showed me. That was really cool.